tools/encode_program_info/encode_program_info.cpp

```diff
@@ -32,22 +32,34 @@ _emit_program_info_file(const char* file_name, const char* symbol_name, uint8_t*
 }
 
 static ebpf_helper_function_prototype_t _ebpf_helper_function_prototype[] = {
-    {1,
-     "bpf_map_lookup_elem",
-     EBPF_RETURN_TYPE_PTR_TO_MAP_VALUE_OR_NULL,
-     {EBPF_ARGUMENT_TYPE_PTR_TO_MAP, EBPF_ARGUMENT_TYPE_PTR_TO_MAP_KEY}},
-    {2,
-     "bpf_map_update_elem",
-     EBPF_RETURN_TYPE_INTEGER,
-     {EBPF_ARGUMENT_TYPE_PTR_TO_MAP, EBPF_ARGUMENT_TYPE_PTR_TO_MAP_KEY, EBPF_ARGUMENT_TYPE_PTR_TO_MAP_VALUE}},
-    {3,
-     "bpf_map_delete_elem",
-     EBPF_RETURN_TYPE_INTEGER,
-     {EBPF_ARGUMENT_TYPE_PTR_TO_MAP, EBPF_ARGUMENT_TYPE_PTR_TO_MAP_KEY}},
-    {4,
-     "bpf_tail_call",
-     EBPF_RETURN_TYPE_INTEGER,
-     {EBPF_ARGUMENT_TYPE_PTR_TO_CTX, EBPF_ARGUMENT_TYPE_PTR_TO_MAP, EBPF_ARGUMENT_TYPE_ANYTHING}}};
+    {
+        1,
+        "bpf_map_lookup_elem",
+        EBPF_RETURN_TYPE_PTR_TO_MAP_VALUE_OR_NULL,
+        {EBPF_ARGUMENT_TYPE_PTR_TO_MAP, EBPF_ARGUMENT_TYPE_PTR_TO_MAP_KEY},
+        0, // flags
```

Comment:
in your other changes you didn't have any such comment.  Suggest removing comment.  If you think you need one then define
`#define EBPF_HELPER_FLAG_NONE 0` or similar and use that.